style/values/specified/image.rs

```diff
@@ -46,7 +46,7 @@ pub type Image = generic::Image<Gradient, SpecifiedUrl, Color, Percentage, Resol
 #[cfg(feature = "gecko")]
 size_of_test!(Image, 16);
 #[cfg(feature = "servo")]
-size_of_test!(Image, 24);
+size_of_test!(Image, 16);
```

Comment:
You can unify these since both Gecko and Servo are 16 now.

Reply:
Done.